### ISSUE TITLE
responding to SIGINT

### ISSUE THREAD

**Issue by euneuber** [qownnotes/qc#1]
Thanks for this great tool!

I'd like to make a small suggestion:

- on `[Ctrl]-C` (SIGINT) an exit code of 128 + signal number = 130 should be returned (see `man 1 bash` in "SHELL GRAMMAR" / "Simple Commands" for reference)
- on `[Ctrl]-C` (SIGINT) do not put any ouput on terminal.

**Comment by pbek** [comment]
Thank you for your kind words (and the first like). 😁 

What are we talking about? What do you want to quit with `Ctrl + C`? `fzf`?

**Comment by euneuber** [comment]
Yes, `qc` starts `fzf` so SIGINT terminates `fzf` and it already responds with exit code 130 (one can start `fzf` alone in a terminal and press `[Ctrl]-C` followed by `echo $?`)

**Comment by pbek** [comment]
And you want that this exit code is passed through by qc when exiting?

**Comment by pbek** [comment]
And what do you want to achieve by this?

**Comment by pbek** [comment]
It should already be returned by:

https://github.com/qownnotes/qc/blob/9cb2c1c41b12fb32db9fa2ddf83d1b22b4734572/cmd/exec.go#L41

But it doesn't seem to be returned by Cobra:

https://github.com/qownnotes/qc/blob/9cb2c1c41b12fb32db9fa2ddf83d1b22b4734572/cmd/exec.go#L14-L20

**Comment by pbek** [comment]
You can play around with it if you like...

**Comment by euneuber** [comment]
Yes, `qc` starts `fzf` so SIGINT terminates `fzf` and it already responds with exit code 130 (one can start `fzf` alone in a terminal and press `[Ctrl]-C` followed by `echo $?`)

**Comment by pbek** [comment]
Let me know if you find a way to pass the exit code on...

**Comment by pbek** [comment]
@euneuber, I wanted to implement this in https://github.com/qownnotes/qc/commit/70361f315412af4b6dd5cfda2b938cec385a0dfe, but even without that commit `qc exec` already returns exit code 130, when I `Ctrl + C` a command (like `sleep 1000`). Did we talk about something different?

**Comment by pbek** [comment]
I will close this issue until there is more information.
